includes/class-amp-theme-support.php

```diff
@@ -2278,6 +2315,15 @@ public static function prepare_response( $response, $args = [] ) {
 					$non_amp_url = add_query_arg( AMP_Validation_Manager::VALIDATION_ERRORS_QUERY_VAR, $blocking_error_count, $non_amp_url );
 				}
 
+				// Remove debugging query args.
+				$non_amp_url = remove_query_arg(
+					[
+						self::DISABLE_RESPONSE_CACHE_QUERY_VAR,
+						AMP_Validation_Error_Taxonomy::REJECT_ALL_VALIDATION_ERRORS_QUERY_VAR,
+					],
```

Comment:
Per above, this could just remove `amp_flags`.

Reply:
Sure, I'll apply this also.

Reply:
Good idea, it's [applied here](https://github.com/ampproject/amp-wp/pull/3448/commits/19363fc23939a38173ecfe0bd4d4baad3de2583a#diff-2585472f207548f626a43a7ff3cab922R2326).